### PR TITLE
Reworked the documentation and the script

### PR DESCRIPTION
**Pull request by lburgey**
- Moved the topic into a subdir
- Added e.g. requirements.txt for the python deps
- Changes to the docs:
  - More to the point
  - Added many missing instructions
- Changes to the script
   - Now sourced instead of eval'd
   - Reduced the exported env vars to a minimum
   - Handle errors 
   - Checks for its dependencies

### CODE REVIEW

File: users/rclone-swift/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+federated Identity Management service EGI Checkin through the OpenID connect protocol.
+
+Follow these instructions:
+1. Clone this repository and change into this directory.
+1. Install [rclone](https://rclone.org/install/)
+1. Install [jq](https://stedolan.github.io/jq/)
+1. Install two python packages:
+	1. `fedcloudclient`, see [here](https://fedcloudclient.fedcloud.eu/install.html)
+	1. `python-openstackclient`, see [here](https://pypi.org/project/python-openstackclient/)
+1. Execute the procedure explained [here](https://docs.egi.eu/users/cloud-compute/auth/#check-in-and-access-tokens) to access and use the EGI Check-in service.
```
Comment (marcvs) [inline users/rclone-swift/README.md]: Can't we leave out this line from the docs entirely?
I mean: The way of handling the RT is insecure, and the next line describes what I think is the perfect alternative.

File: users/rclone-swift/rclone-swift-env [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+if [[ -z $OIDC_AGENT_ACCOUNT ]]; then
+	echo "You have not specified the oidc-agent account to be used. You can specify it with '--oa-account'  (or via export OIDC_AGENT_ACCOUNT)"
+    return 4
+fi
+export OIDC_AGENT_ACCOUNT
+
+# unlock oidc-agent
+echo "Using OIDC Agent Account: $OIDC_AGENT_ACCOUNT"
+eval "$(oidc-keychain --accounts ${OIDC_AGENT_ACCOUNT})" >/dev/null
```
Comment (marcvs) [inline users/rclone-swift/rclone-swift-env]: Ok; for the record "Marcus ist zu blöd den PR zu verändern"
Please replace line 61 with this: (rationale: oidc-keychain is not supported)
```
eval "$(oidc-agent-service use )" 
oidc-add -a | grep -q ^${OIDC_AGENT_ACCOUNT}$ || {
    oidc-add ${OIDC_AGENT_ACCOUNT}
}
```

**Comment by marcvs** [comment]
General comment: Much nicer than before!
The Pipfile... does it want to suggest, that this is going to be available via pip??

**Comment by lburgey** [comment]
> The Pipfile... does it want to suggest, that this is going to be available via pip??

No, like the requirements.txt it can be used to install the two python depenencies with one command. Its not important, but i thought it could be useful

File: users/rclone-swift/README.md (hunk 1 above)
Comment (lburgey) [reply users/rclone-swift/README.md]: Sure, i'll remove it

Comment (marcvs) [reply users/rclone-swift/README.md]: Wait, but what I meant was only the reference to the one line of doc that sends to the "aai.egi.eu/fedcloud"  endpoint.

Comment (lburgey) [reply users/rclone-swift/README.md]: Do you mean line 19? I'm not sure if i got you right

File: users/rclone-swift/rclone-swift-env (hunk 2 above)
Comment (lburgey) [reply users/rclone-swift/rclone-swift-env]: prep-rclone.sh already used `oidc-keychain`. What do you mean by oidc-keychain is not supported? Isn't it always bundled with oidc-agent?

Comment (lburgey) [reply users/rclone-swift/rclone-swift-env]: The oidc-agent version in Arch linux does not yet ship `oidc-agent-service` -_-

File: users/rclone-swift/README.md (hunk 1 above)
Comment (marcvs) [reply users/rclone-swift/README.md]: This is the only line to go away in this context
```
Execute the procedure explained [here](https://docs.egi.eu/users/cloud-compute/auth/#check-in-and-access-tokens) to access and use the EGI Check-in service
```

File: users/rclone-swift/rclone-swift-env (hunk 2 above)
Comment (marcvs) [reply users/rclone-swift/rclone-swift-env]: Oh Arch linux... Oh god. Thanks for pointing that one out. I've contacted the guy doing that.

Comment (marcvs) [reply users/rclone-swift/rclone-swift-env]: Well, for me (and other users of `oidc-agent-service`) `oidc-keychain` breaks things but using a new agent instance. We could either fix the (externally supplied) `oidc-keychain` tool, (which mimmicks a certain behaviour; my fix would almost certainly break that) or just move to keychain..

Comment (lburgey) [reply users/rclone-swift/rclone-swift-env]: This is done, see latest commit

File: users/rclone-swift/README.md (hunk 1 above)
Comment (lburgey) [reply users/rclone-swift/README.md]: Done

**Comment by lburgey** [comment]
@marcvs I think I addressed all the comments. Can we merge this?

**Review by marcvs — COMMENTED** [review]
:+1:

**Review by marcvs — COMMENTED** [review]
We've discussed the changes, and I see them implemented.

**Review by marcvs — COMMENTED** [review]
Topp!

File: README.md [hunk 3]
```diff
@@ -4,6 +4,6 @@ This repository hold the EOSC-Synergy public documentation.
 
 ## User manuals and tutorials
```
Comment (marcvs) [inline README.md:5]: Gut!

**Review by marcvs — APPROVED** [review]
Topp

**Pull request merged**